README.md

```diff
@@ -29,7 +29,7 @@ After setup, it is recommended you update this README to describe your custom im
 
 The easiest way to start customizing is by looking at and modifying `config/recipe.yml`. It's documented using comments and should be pretty easy to understand.
 
-If you want to add custom configuration files, you can just add them in the `/usr/etc/` directory, which is the official OSTree "configuration template" directory and will be applied to `/etc/` on boot. `usr` is copied into your image by default. If you need to add other directories in the root of your image, that can be done in the Containerfile. Writing to `/var/` in the image builds of OSTree-based distros isn't supported and will not work, as that is a local user-managed directory!
+If you want to add custom configuration files, you can just add them in the `/usr/etc/` directory (under `/config/files`), which is the official OSTree "configuration template" directory and will be applied to `/etc/` on boot. `usr` is copied into your image by default. If you need to add other directories in the root of your image, that can be done in the Containerfile. Writing to `/var/` in the image builds of OSTree-based distros isn't supported and will not work, as that is a local user-managed directory!
```

Comment:
Maybe here too I would have changed the part
"`usr` is copied into your image..." to "`config/files/usr` is copied into your image" because once again `/usr/etc/` with the preceding `/` is not the source (`config/files/...`) but the destination.

Reply:
good point